app/controllers/api/admin/users_controller.rb

```diff
@@ -7,17 +7,20 @@
 
 class Api::Admin::UsersController < Api::Admin::AdminController
 
-  def toggle_admin
+  def update_role
```

Comment:
move to separate controller: api/admin/users/role_controller.rb
action: #update 
http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/

Reply:
Done